server/src/com/thoughtworks/go/server/cache/ArtifactCache.java

```diff
@@ -24,11 +24,13 @@
 import java.util.concurrent.ConcurrentSkipListSet;
 
 import com.thoughtworks.go.server.service.ArtifactsDirHolder;
+import org.apache.log4j.Logger;
```

Comment:
Also refer comments on https://github.com/gocd/gocd/pull/2452
For testing the log messages.